src/main/java/net/jeqo/gizmo/Managers/ConfigManager.java

```diff
@@ -0,0 +1,68 @@
+package net.jeqo.gizmo.Managers;
```

Comment:
Very nice config manager, although as there is a repository codebase restructure in progress, this will now be a part of the new utilities package that will soon be published publicly 😄